### DISCUSSION TITLE
classes/class_animationmixer

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#280] [User-contributed notes]
# classes/class_animationmixer

Inherits: Node< Object Inherited By: AnimationPlayer, AnimationTree Base class for AnimationPlayer and AnimationTree. Description: Base class for AnimationPlayer and AnimationTree to manage animati...

https://docs.godotengine.org/en/stable/classes/class_animationmixer.html

**Comment by tangx246** [comment]
It looks like `animation_started` signal is not called for looping animations just like in `animation_finished`, but it is not documented in that case while it is documented in `animation_finished`

**Reply by RobertHDPotatoes** [reply]
Can confirm. Was just ripping my hair out on why only certain animations were emitting the signals and not others. In my case, I am using an AnimationTree but not sure if that matters.

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot/pull/108615

**Reply by adam3esamkk-wq** [reply]
Jddjjfn
